alibi_detect/cd/base_online.py

```diff
@@ -290,7 +377,92 @@ def _configure_ref(self):
     def _update_state(self, x_t: np.ndarray):
         pass
 
+    def _set_state_path(self, dirpath: Union[str, os.PathLike]):
```

Comment:
What's the difference between `BaseUniDriftOnline` and `BaseMultiDriftOnline` that means one has `_set_state_path` and the other doesn't?

Reply:
Thanks, that shouldn't exist. I renamed it `_set_state_dir` but somehow didn't delete the old one.

Reply:
Removed in [5dea156](https://github.com/SeldonIO/alibi-detect/pull/604/commits/5dea1566bff351cdeb5865f5c612d60817f5cd40).